hwcomposer/hwcomposer.pro

```diff
@@ -27,7 +27,21 @@ HEADERS += hwcomposer_backend_v10.h
 SOURCES += hwcomposer_backend_v11.cpp
 HEADERS += hwcomposer_backend_v11.h
 
+versionAtLeast(QT_MINOR_VERSION, 8) {
+QT += core-private gui-private egl_support-private waylandcompositor-private dbus sensors fontdatabase_support-private eventdispatcher_support-private theme_support-private
```

Comment:
please fix the indentation. also, remove 'sensors' here since it's conditionally appended on line 38.

Reply:
Adjusted